tools/integration_tests/read_cache/disabled_cache_ttl_test.go

```diff
@@ -96,8 +96,8 @@ func TestDisabledCacheTTLTest(t *testing.T) {
 
 	// Define flag set to run the tests.
 	flagsSet := [][]string{
-		{"--implicit-dirs", "--stat-cache-ttl=0s", "--config-file=" + createConfigFile(cacheCapacityInMB, false, configFileName, false)},
-		{"--stat-cache-ttl=0s", "--config-file=" + createConfigFile(cacheCapacityInMB, false, configFileNameForParallelDownloadTests, true)},
+		{"--implicit-dirs", "--stat-cache-ttl=0s", "--config-file=" + createConfigFile(cacheCapacityInMB, false, configFileName, false, getDefaultCacheDirPathForTests())},
+		{"--stat-cache-ttl=0s", "--config-file=" + createConfigFile(cacheCapacityInMB, false, configFileNameForParallelDownloadTests, true, getDefaultCacheDirPathForTests())},
```

Comment:
are we explicitly not using ramCache here and in other tests?

Reply:
Yes, the intention of this test is just to ensure that file-cache with and w/o parallel downloads work with ram based storage system, so we don't need to add in all the tests.